src/DI.Specification.Tests/Fakes/ClassWithAmbiguousCtorsAndAttribute.cs

```diff
@@ -0,0 +1,26 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace Microsoft.Extensions.DependencyInjection.Specification.Fakes
+{
+    public class ClassWithAmbiguousCtorsAndAttribute
+    {
+        public ClassWithAmbiguousCtorsAndAttribute(string data)
+        {
+            CtorUsed = "string";
+        }
+
+        [ActivatorUtilitiesConstructor]
+        public ClassWithAmbiguousCtorsAndAttribute(IFakeService service, string data)
+        {
+            CtorUsed = "IFakeService, string";
+        }
+
+        public ClassWithAmbiguousCtorsAndAttribute(IFakeService service,  IFakeService service2, string data)
```

Comment:
Extra space

Reply:
Also, use a different service type. It's super unusual to do something like this.